include/moves.h

```diff
@@ -359,6 +359,7 @@ enum
     MOVE_DOOM_DESIRE,
     MOVE_PSYCHO_BOOST,
 };
+extern const u8 gMoveNames[][13];
```

Comment:
Shouldn't it be externed in another header? I beilieve it's data2.h in pokeruby?

Reply:
🤷‍♂️

Reply:
In decompiling scrcmd I've made that change; when merging that in I'll take this out.